File: snap-crescent/src/main/java/com/codeinsight/snap_crescent/thumbnail/ThumbnailRepository.java
package com.codeinsight.snap_crescent.thumbnail;

import org.springframework.stereotype.Repository;

import com.codeinsight.snap_crescent.common.BaseRepository;

@Repository
public class ThumbnailRepository extends BaseRepository<Thumbnail>{
			
			public ThumbnailRepository() {
				super(Thumbnail.class);
			}

}

File: snap_crescent_mobile-flutter/lib/repository/thumbnail_repository.dart
import 'package:snap_crescent/repository/base_repository.dart';

class ThumbnailRepository extends BaseRepository{

  static final _tableName = 'THUMBNAIL'; 

  ThumbnailRepository._privateConstructor():super(_tableName);
  static final ThumbnailRepository instance = ThumbnailRepository._privateConstructor();
  
}